### ISSUE TITLE
Image build failing.

### ISSUE THREAD

**Issue by andzajan** [hyoo/alphafold_singularity#1]
Hi, I was getting error "source" not found while trying to build an image using Singularity 3.8.1.

Problem was solved by changing line 14 (https://github.com/hyoo/alphafold_singularity/blob/main/alphafold.def#L14) to:

`. /opt/miniconda3/etc/profile.d/conda.sh`

**Comment by hyoo** [comment]
This is related to the shell. Are you using bash? 
https://askubuntu.com/questions/25488/what-is-the-difference-between-source-and

**Comment by andzajan** [comment]
Yes, building from Ubuntu 20.04 and I guess that builder is executing "source" on the host OS, so I don't know why during my build it's not using bash. I am not very experienced with Singularity.

**Comment by aeszter** [comment]
I've encountered the same error when building on openSUSE, where `/bin/sh` is symlinked to `bash`. So maybe source was called inside the container after all?

**Comment by fredricj** [comment]
The default shell in singularity is /bin/sh, i.e. bash in posix mode. Changing `%post` to `%post -c /bin/bash` switches to using bash instead which would fix this error and also prevent other errors with non-posix syntax in other shell scripts used by conda
